Add more use cases for validation script, improve checks (#157)
packit/deployment: cron-jobs/validation/README.md

[packit-service-validation.py](./packit-service-validation.py) - script in the image

> The script verifies building Copr builds and running Testing farm for pull requests in `packit/hello-world` repo:

Reviewer: ```suggestion
The script verifies that Copr builds and Testing farm runs are processed correctly for pull requests in `packit/hello-world` repo:
```
this spelling is more clear to me, is it okay?

Author: of course, will change
